src/bundler/bundler.rs

```diff
@@ -1,5 +1,37 @@
+use std::str::FromStr;
+
+use clap::Parser;
+use ethereum_types::U256;
+use ethers::types::Address;
+
 use crate::models::wallet::Wallet;
 
+#[derive(Debug, Parser, PartialEq)]
+pub struct BundlerOpts {
```

Comment:
I think this should be only Opts, we using full import path in the bundler.rs: aa_bundler::bundler::bundler::Opts